modules/ref_fb_module/src/ref_fb_module_impl.cpp

```diff
@@ -3,7 +3,9 @@
 #include <ref_fb_module/classifier_fb_impl.h>
 #include <ref_fb_module/power_fb_impl.h>
 #include <ref_fb_module/ref_fb_module_impl.h>
+#ifdef ENABLE_RENDERER
```

Comment:
I'd prefer to use `#if defined(ENABLE_RENDERER)` as it can be extended and chained
e.g `#if defined(ENABLE_RENDERED) && defined(SFM_AVAILABLE)`